frontend/src/utils/getRoutes.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "node:fs";
+import path from "node:path";
+
+// Helper function to list all paths recursively
+function listPaths(dir: string): string[] {
```

Comment:
This is a nice dynamic solution instead of hard coding. We should add a test for the files in this function.